jscomp/bsb/templates/react/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "${bsb:name}",
+  "private": true,
+  "version": "${bsb:proj-version}",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "bsb -make-world -w",
+    "build": "webpack -w",
+    "clean": "bsb -clean-world"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "reason-react": ">=0.1.4"
```

Comment:
I wanted the new reason-react release to be timed with yours. I can release it an hour before the new bs stable release? It's circular because the new rr example projects need the new bs